pkg/proxy/upstream.go

```diff
@@ -85,9 +85,13 @@ func (r *upstreamRequest) OnDestroyStream() {}
 
 func (r *upstreamRequest) endStream() {
 	upstreamResponseDurationNs := time.Now().Sub(r.startTime).Nanoseconds()
+
 	r.host.HostStats().UpstreamRequestDuration.Update(upstreamResponseDurationNs)
+	r.host.HostStats().UpstreamRequestDurationEWMA.Update(upstreamResponseDurationNs)
```

Comment:
I think `EWMA` is a lb extension that the core framework should not be aware of.

Reply:
my mistake，lgtm